opensrp-reporting/src/main/java/org/smartregister/reporting/dao/ReportIndicatorDaoImpl.java

```diff
@@ -142,8 +144,8 @@ protected void saveTallies(Map<String, IndicatorQuery> indicatorQueries, Map.Ent
                 if (!executedQueries.contains(queryString)) {
                     Timber.i("QUERY : %s", queryString);
                     float count = executeQueryAndReturnCount(queryString, database);
-
-                    if (count > 0) {
+                    boolean shouldAllowZeroTallies = Utils.getBooleanProperty(Constants.ReportingConfig.SHOULD_ALLOW_ZERO_TALLIES);
```

Comment:
Can we have the Sample app have the implementation for this config as well? 
It'll be helpful in showcasing how this property is set

Reply:
cool